src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/EventLogDataCollector.cs

```diff
@@ -566,16 +566,17 @@ private void ConfigureMaxEntries(CollectorNameValueConfigurationManager collecto
         }
     }
 
+    [return: NotNull]
     private EventLogSessionContext GetEventLogSessionContext(DataCollectionContext dataCollectionContext)
     {
-        EventLogSessionContext eventLogSessionContext;
+        EventLogSessionContext? eventLogSessionContext;
         bool eventLogContainerFound;
         lock (ContextMap)
         {
             eventLogContainerFound = ContextMap.TryGetValue(dataCollectionContext, out eventLogSessionContext);
         }
 
-        if (!eventLogContainerFound)
+        if (eventLogContainerFound == false)
```

Comment:
Previous code was ok to me. Usually when I see `== false`, I expect some nullable bool which isn't the case here.